recipe/activate.sh

```diff
@@ -23,6 +23,17 @@ else
     CUDA_LDFLAGS="${CUDA_LDFLAGS} -L${CONDA_PREFIX}/${targetsDir}/lib/stubs"
 fi
 
+# For conda-build we add the host requirements prefix to the include and link
+# paths for nvcc because it is separate from the build prefix where nvcc is
+# installed.
+if [ "${CONDA_BUILD:-0}" = "1" ]; then
+
+    NVCC_PREPEND_FLAGS="${NVCC_PREPEND_FLAGS} -I${PREFIX}/${targetsDir}/include"
+    NVCC_PREPEND_FLAGS="${NVCC_PREPEND_FLAGS} -L${PREFIX}/${targetsDir}/lib"
+    NVCC_PREPEND_FLAGS="${NVCC_PREPEND_FLAGS} -L${PREFIX}/${targetsDir}/lib/stubs"
+    export NVCC_PREPEND_FLAGS
```

Comment:
Should we move these into the block above (along with the comment here)?

Reply:
No. The first block is to augment calls to the host compiler. This block is for the cuda compiler.

Reply:
Maybe I should move this block down so it doesn't separate the definition and use of `CUDA_CFLAGS`.

Reply:
Also, NVCC_PREPEND_FLAGS is already modified below, so I need to harmonize my new addtion with that.

Reply:
Yeah that makes sense actually. Moving next to `NVCC_PREPEND_FLAGS` is better. Thanks! 🙏